wgengine/magicsock/magicsock.go

```diff
@@ -927,6 +953,10 @@ func (c *Conn) determineEndpoints(ctx context.Context) ([]tailcfg.Endpoint, erro
 		addAddr(ap, tailcfg.EndpointExplicitConf)
 	}
 
+	for _, ap := range c.staticEndpoints {
```

Comment:
This would eventually replace the setting of endpoints from `TS_DEBUG_PRETENDPOINTS` above ^ but should first verify that the configfile works for use cases we want to support